test/Microsoft.ServiceFabric.Actors.Tests/AssemblyInfo.cs

```diff
@@ -3,6 +3,12 @@
 // Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
+using Microsoft.ServiceFabric.Services.Remoting;
```

Comment:
Double-check using directives in all .cs files modified in this pull request and remove the unused directives.